charts/node/values.yaml

```diff
@@ -54,58 +89,104 @@ ingress:
   #    hosts:
   #      - chart-example.local
 
-# Add extra labels on pods and services
-#extraLabels:
-#  type: rpc
 
+## Deploy a Polkadot node
+## ref: https://wiki.polkadot.network/docs/maintain-sync
```

Comment:
Should probably refer to substrate rather than polkadot given that is the generic upstream product?